src/pudl/analysis/__init__.py

```diff
@@ -11,6 +11,7 @@
     ferc1_eia_record_linkage,
     mcoe,
     plant_parts_eia,
+    record_linkage,
```

Comment:
Just making a note that this module name is pretty close to `recordlinkage` which is a dependency in the FERC to EIA match, but I'm planning on removing the `recordlinkage` dependency there, so it's fine.